shared/src/error/mod.rs

```diff
@@ -19,13 +19,18 @@ pub enum ExitCode {
     /// Indicates failure to find an actor in the state tree.
     SysErrSenderInvalid = 1,
 
-    /// Indicates failure to find the code for an actor.
+    /// Indicates that the message sender was not in a valid state to send this message. This means
```

Comment:
Nit that "the message shouldn't have been included on-chain" isn't strictly true -- a sender's balance can change within tipset execution causing not enough funds to cover message gas.

Also note that the nonce case never actually happens, since blocks containing such messages are invalid (and so not on chain).

Reply:
Fixed.